rotkehlchen/externalapis/etherscan.py

```diff
@@ -285,11 +289,56 @@ def get_transactions(
             for entry in result:
                 gevent.sleep(0)
                 try:
-                    tx = deserialize_ethereum_transaction(  # type: ignore
-                        data=entry,
-                        internal=is_internal,
-                        ethereum=None,
-                    )
+                    # Handle genesis block transactions
+                    if is_genesis_tx(entry) is False:
+                        tx = deserialize_ethereum_transaction(  # type: ignore
+                            data=entry,
+                            internal=is_internal,
+                            ethereum=None,
+                        )
+                    else:
+                        # Handling genesis transactions
+                        if self.db is None:
+                            raise DeserializationError('DB not initialized')
+                        dbtx = DBEthTx(self.db)
```

Comment:
All this logic should be in `DBEthTx()` and be called `get_or_create_genesis_transaction`